meinberlin/assets/scss/components/_dialogue_box.scss

```diff
@@ -1,4 +1,32 @@
+.dialogue-box__container {
+    position: relative;
+}
+
+.dialogue-box__background-left,
+.dialogue-box__background-right {
+    position: absolute;
+    height: 94px;
+    background-image: url('/static/images/dialoguebox_border.png');
+    background-repeat: repeat-x;
+    bottom: 0;
+    z-index: 10;
+    width: 15%;
+
+    @media screen and (min-width: $wrapper-width + 10rem) {
+        width: 20%;
+    }
+}
+
+.dialogue-box__background-left {
+    left: 0;
+}
+
+.dialogue-box__background-right {
+    right: 0;
+}
+
 .dialogue-box {
+    z-index: 100;
```

Comment:
Does the userdropdown still go over it? I think 100 is a bit too high, isn't 1 sufficient?

Reply:
forgot about user dropdown, 1 wasnt sufficient but i'll check this, dropdowns are default very high normally.